### PR TITLE
order elements in move2_loc - analyzer.R

### PR DESCRIPTION
**Pull request by andreakoelzsch**

### CODE REVIEW

**Comment by andreakoelzsch** [comment]
Hi @cleemansen, we need the elements in the cargo agent for move2_loc to be ordered not alphabetically. I have done this by simply adding A, B, C to the names that will (hopefull) appear and the ordering is happening on. If this is correct, and leads to what we need, please pipe through to MoveApps (that is still necessary, or?)

**Comment by cleemansen** [comment]
Hello @andreakoelzsch .

1. are `A.positions_bounding_box`, `H.taxa` and others valid R argument names (dots in names)?
2. at least it is invalid for JSON (these names gets serialized into a json document)
3. you have to use the same names on the calling side, too (eg L70ff)
4. the unit tests are failing
5. MoveApps presents the JSON file as it is. So to avoid iteration inspect a generated json file. I hope some unit tests are producing an output json file. If not, I'm happy to support.
6. yes - I have to insert the next cargo-agent into MoveApps (at least I think so)

**Comment by andreakoelzsch** [comment]
Thanks for the comments @cleemansen!
I have updated so that now all names are new. The addition has changed from A.** to A_**. I hope that json allows that? unit tests are successful now, at least in RStudio

File: src/analyzer/move2_move2_loc/analyzer.R [hunk 1]
```diff
@@ -106,26 +106,28 @@ mapMove2Move2_locOutput <- function(
...
-    positions_total_number = positions_total_number,
-    animals_total_number = animals_total_number,
-    animal_names = animal_names,
-    animal_attributes = animal_attributes,
-    taxa = taxa,
-    tracks_total_number = tracks_total_number,
-    track_names = track_names,
-    number_positions_by_track = number_positions_by_track,
-    track_attributes = track_attributes
+    list(positions_bounding_box = positions_bounding_box),
```
[the remark this answers is not in the document]
Comment (cleemansen) [reply src/analyzer/move2_move2_loc/analyzer.R:118]: kudos: https://stackoverflow.com/a/32102277/810944

**Comment by andreakoelzsch** [comment]
@cleemansen danke für den order-array fix! dann baue ich hier nochmal um, dass das A, B_, ... verschwindet.

**Review by cleemansen — APPROVED** [review]

**Comment by cleemansen** [comment]
> @cleemansen danke für den order-array fix! dann baue ich hier nochmal um, dass das A, B_, ... verschwindet.

Das habe ich direkt miterledigt @andreakoelzsch !

**Comment by andreakoelzsch** [comment]
@cleemansen moment!

**Comment by andreakoelzsch** [comment]
die ordnung im object so wie es jetzt ist, würde ich nochmal anpassen...

**Comment by cleemansen** [comment]
Scheint jetzt alles ready zu sein @andreakoelzsch . Soll ich mich um das Release + Integration in MoveApps kümmern?

**Comment by andreakoelzsch** [comment]
ja bitte gern @cleemansen . danke!!

**Pull request merged**